CHANGELOG.rst

```diff
@@ -1,5 +1,10 @@
 .. _changelog:
 
+0.7.4
+-----
+
+This is a patch release that includes a notebooks change and some improvements to the Knowledge Graph.
```

Comment:
I'd change the sentence to `includes a change in notebooks` (or remove the `a` article). My English is not top-notch, this may not be really necessary :nerd_face:

Reply:
right, the notebooks have not really changed is the version of them that changed in the release.
I'll leave it as is for the moment, we can update later if needed.